MAUI/DataGrid/Row-height-Customization.md

```diff
@@ -0,0 +1,153 @@
+
+To customize header row height, follow the code example:
+
+{% highlight c# %}
+//Customizing row height in SfDataGrid
+dataGrid.RowHeight = 60;
+{% endhighlight %}
+
+## QueryRowHeight
+
+The [SfDatagrid.QueryRowHeight](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_QueryRowHeight) event returns row heights on demand. This event receives two arguments, namely the sender handles the data grid, and the [DataGridQueryRowHeightEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridQueryRowHeightEventArgs.html). The `DataGridQueryRowHeightEventArgs` has the following properties:
+
+ * [RowIndex](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridQueryRowHeightEventArgs.html#Syncfusion_Maui_DataGrid_DataGridQueryRowHeightEventArgs_RowIndex): This property helps to identify a particular row in the grid.
+ * [Height](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridQueryRowHeightEventArgs.html#Syncfusion_Maui_DataGrid_DataGridQueryRowHeightEventArgs_Height): This property sets and returns the height of a grid row on demand. Default line size of the rows is 47.
+ * `Handled`: This property decides whether the specified height can be set to the row or not. The default value is `false`. When this property is not set, the decided height is not set to the row.
+
+To hook the `SfDatagrid.QueryRowHeight` event, and customize height of a row, follow the code example:
+
+{% highlight c# %}
+//Hooks QueryRowHeight event in SfDataGrid
+dataGrid.QueryRowHeight += DataGrid_QueryRowHeight;
+
+//Event to set the row height on demand
+private void DataGrid_QueryRowHeight(object sender, DataGridQueryRowHeightEventArgs e)
+{
+    //Sets height of the fifth row
+    if (e.RowIndex == 5)
+    {
+        e.Height = 100;
+        e.Handled = true;
+    }
+} 
+{% endhighlight %}
+
+The data grid allows you to customize the height of grid rows with various customizing options while auto calculating the row height based on the content.
+
+## Auto fit the grid rows based on content
+
+The data grid supports the AutoRowHeight feature. The height of a row can be customized based on the content. This can be achieved using the `SfDatagrid.QueryRowHeight` event and the [SfDatagrid.GetIntrinsicRowHeight](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridQueryRowHeightEventArgs.html#Syncfusion_Maui_DataGrid_DataGridQueryRowHeightEventArgs_GetIntrinsicRowHeight_System_Int32_System_Boolean_System_Collections_Generic_List_System_String__) method. The `SfDatagrid.QueryRowHeight` event returns the row height on demand. The `SfDatagrid.GetIntrinsicRowHeight` method returns the height of the row based on the content.
```

Comment:
Replace the para with below,

The height of a row can be customized based on its cell content by using the `SfDatagrid.QueryRowHeight` event and the [SfDatagrid.GetIntrinsicRowHeight](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridQueryRowHeightEventArgs.html#Syncfusion_Maui_DataGrid_DataGridQueryRowHeightEventArgs_GetIntrinsicRowHeight_System_Int32_System_Boolean_System_Collections_Generic_List_System_String__) method. The `SfDatagrid.QueryRowHeight` event returns the row height on demand. The `SfDatagrid.GetIntrinsicRowHeight` method returns the height of the row based on the content.

Reply:
Replaced the Paragraph